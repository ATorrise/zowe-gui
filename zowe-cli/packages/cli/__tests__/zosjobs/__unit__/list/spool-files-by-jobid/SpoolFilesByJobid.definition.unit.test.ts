/*
* This program and the accompanying materials are made available under the terms of the
* Eclipse Public License v2.0 which accompanies this distribution, and is available at
* https://www.eclipse.org/legal/epl-v20.html
*
* SPDX-License-Identifier: EPL-2.0
*
* Copyright Contributors to the Zowe Project.
*
*/

import { ICommandDefinition } from "@zowe/imperative";

describe("zos-jobs list spool-files-by-jobid definition", () => {
    it("should not have changed", () => {
        const definition: ICommandDefinition
            = require("../../../../../src/zosjobs/list/spool-files-by-jobid/SpoolFilesByJobid.definition").SpoolFilesByJobidDefinition;
        delete definition.handler;
        expect(definition).toBeDefined();
        expect(definition).toMatchSnapshot();
    });
});
