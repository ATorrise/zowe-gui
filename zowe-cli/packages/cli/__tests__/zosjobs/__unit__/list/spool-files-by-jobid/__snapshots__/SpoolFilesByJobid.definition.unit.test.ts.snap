// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`zos-jobs list spool-files-by-jobid definition should not have changed 1`] = `
Object {
  "aliases": Array [
    "sfbj",
  ],
  "description": "Given a z/OS job JOBID, list the spool files (DDs) for a z/OS job on the JES/spool queues. The command does not pre-validate the JOBID. The command presents errors verbatim from the z/OSMF Jobs REST endpoints.",
  "examples": Array [
    Object {
      "description": "List the spool files of the job with JOBID JOB00123",
      "options": "job00123",
    },
  ],
  "name": "spool-files-by-jobid",
  "outputFormatOptions": true,
  "positionals": Array [
    Object {
      "description": "The z/OS JOBID of the job with the spool files you want to list. No pre-validation of the JOBID is performed.",
      "name": "jobid",
      "required": true,
      "type": "string",
    },
  ],
  "profile": Object {
    "optional": Array [
      "zosmf",
    ],
  },
  "summary": "List spool files of a z/OS job",
  "type": "command",
}
`;
